### ISSUE TITLE
Hangfire 1.7.6 compatibility with .NET core 3.0?

### ISSUE THREAD

**Issue by arlingan** [HangfireIO/Hangfire#1528]
I was trying to find the documentation about it but couldn't find it anywhere. I noticed Hangfire 1.7.6 is compatible with .NET Standard 2.0. Can someone confirm if this has been tried or tested with .NET core 3.0 apps?

**Comment by Grey007** [comment]
I'm also having issues at the moment with Hangfire after upgrading from 2.2 to 3.0

**Comment by kYann** [comment]
@Grey007 Do you have this issue ?

`System.MissingMethodException: Method not found: 'Microsoft.AspNetCore.Hosting.IHostingEnvironment Microsoft.AspNetCore.Hosting.WebHostBuilderContext.get_HostingEnvironment()'.`

**Comment by Grey007** [comment]
Nope. My issue was related to the docker image I was using. The base .net core 3.0 SDK docker image does not work for the connection to a SQL DB. I had to use the 3.0-Bionic docker image to get it to work.

**Comment by sabiland** [comment]
I have no issues with Hangfire after upgrading to Net Core 3.0.

**Comment by Grey007** [comment]
Are you running through Docker? Because running locally or on a VM it works perfectly.

Kind Regards,
Graham Nelson

________________________________
